packages/inventory-compliance/src/ComplianceRemediationButton/ComplianceRemediationButton.js

```diff
@@ -58,9 +59,15 @@ class ComplianceRemediationButton extends React.Component {
         });
     }
 
+    notEmptyData = async () => {
+        const data = await this.dataProvider();
```

Comment:
Observation: Data processing would be called twice, once to check non-empy values and second to pass the data to via dataProvider to the button. Considering the amount of data flowing through it, it might decrease the performance of the page.

Ideally it might be precomputed/cached or handled down the stream within `RemediationButton`.

Reply:
I am aware of this. But for now, till we have time to refactor, this is the way.